.github/workflows/config-schema.yml

```
@@ -1,56 +1,56 @@
-
-      - name: Set up Python
-        uses: actions/setup-python@v5
-        with:
-          python-version: "3.x"
-
-      - name: Install check-jsonschema
-        run: pip install check-jsonschema
-
-      - name: Validate config.example.yaml against schema
-        run: check-jsonschema --schemafile config-schema.json config.example.yaml
+name: Validate JSON Schema
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)